kvmagent/kvmagent/plugins/bmv2_gateway_agent/volume/helper.py

```
@@ -90,7 +90,7 @@ def _check_rbd_image(image_path):
         """
         check if rbd image exists
         """
-        _s, _ = commands.getstatusoutput('rbd info %s' % image_path)
+        _s, _ = subprocess.getstatusoutput('rbd info %s' % image_path)
```

Ruff (0.15.2)

[error] Starting a process with a shell, possible injection detected

(S605)